index.html

```diff
@@ -274,6 +275,41 @@ <h1 class="underlined">About Me</h1>
                 </ul>
             </div>
         </section>
+
+        <!-- contact me  -->
+        <section class="get-in-touch">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form action="https://formspree.io/f/xyyawvzk" enctype="multipart/form-data" method="POST"
+                class="get-in-touch-form">
+                <div class="name-input">
+                    <input type="text" name="first name" placeholder="First Name" maxlength="30" required>
+                    <input class="last-name-input" type="text" name="last name" placeholder="Last Name" maxlength="30"
+                        required>
+                </div><input type="email" name="email" placeholder="Email Address" required>
+                <textarea name="message" id="message" cols="30" rows="5" maxlength="500" placeholder="Enter text here"></textarea>
+                <button type="submit">Get in touch</button>
```

Comment:
As specified by the previous reviewer, please make sure a user can `submit` a form by clicking on the `submit` button, currently anytime a user clicks the `submit` button they get an error. See details of the error below
<img width="685" alt="Screenshot 2023-03-30 at 20 50 59" src="https://user-images.githubusercontent.com/107192076/228962232-3364dd80-1876-460f-af8e-22e5bf21d5f3.png">